client/lib/users/store.js

```diff
@@ -13,12 +13,13 @@ var Dispatcher = require( 'dispatcher' ),
 	emitter = require( 'lib/mixins/emitter' ),
 	deterministicStringify = require( 'lib/deterministic-stringify' );
 
-var _fetchingUsersByNamespace = {}, // store fetching state (boolean)
-	_usersBySite = {}, // store user objects
-	_totalUsersByNamespace = {}, // store total found for params
-	_usersFetchedByNamespace = {}, // store fetch progress
-	_offsetByNamespace = {}, // store fetch progress
-	_userIDsByNamespace = {}; // store user order
+var _fetchingUsersByNamespace = {},        // store fetching state (boolean)
```

Comment:
This looks a bit off compared to the rest, but looks :+1: when tab space is set to 4.